packages/components/Button/src/Button.types.ts

```diff
@@ -112,6 +112,7 @@ export interface IButtonProps extends Omit<IPressableProps, 'onPress'> {
 
 export interface IButtonSlotProps {
   root: React.PropsWithRef<IViewProps>;
+  ripple?: PressableProps;
```

Comment:
Is Ripple an android-specific slot? At the very least, I'd suggest adding a comment that tells people this is an android-specific slot 

It feels a bit strange to me that we have two android-specific files (Button.android.tsx and Button.settings.android.ts), but  for this one we don't. I understand it's a small addition, but should it be consistent?

Reply:
`<Pressable />` is used for ripple which is android-specific. Though adding a comment wont hurt 😅, so doing that. 

_Button.android.tsx_ and _Button.settings.android.tsx_ exist because `Platform.select` would affect the file size and readability, whereas having `ripple?` in types file is a very insignificant change to have a separate file.

Reply:
When bundling, metro should be smart enough to strip the `Platform.Select` from the bundles so it would not add to file size unless I'm mistaken

Reply:
@tamasane I think Sammy's question is about ripple? is fair, and should be a principle that we decide on for the repo. Should we put platform specific props in their own files, or should be put them in a common file and just leave comments. It may be ok with just one platform specific property, but what about two? Or three? Or more? I'm ok with the comment for now, but I could see pulling these out into platform specific files if we start to see more platform-specific types/props.

Reply:
Yes, that's more where my question was headed (sorry for being vague). This instance should be fine, but once additional platform specific props are needed within the repo, I'd like us to all agree on a organization principle on where to add these. As EJ said stated above